spec/GraphQLOverHTTP.md

```diff
@@ -129,6 +145,8 @@ A request for execution should contain the following request parameters:
 * {variables} - (*Optional*): Values for any Variables defined by the Operation.
 * {extensions} - (*Optional*): This entry is reserved for implementors to extend the protocol however they see fit.
 
+Note: Be aware that `query` is a misleading name as it can contain a string describing the operation that may be a query, a mutation or a subscription.
```

Comment:
```suggestion
Note: Be aware that `query` is a misleading name as it can contain a string describing multiple operations, each of which may be a query, mutation or subscription. A better name would have been `document`, but the term `query` is well established.
```